src/maskAnalysis/utils/getMbrFromPoints.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  difference,
+  normalize,
+  Point,
+  rotate,
+} from '../../utils/geometry/points';
+import { Mbr } from '../getMbr';
+
+/**
+ * Get the four corners of the minimun bounding rectangle from a set of points defining a simple convex polygon.
+ * https://www.researchgate.net/profile/Lennert_Den_Boer2/publication/303783472_A_Fast_Algorithm_for_Generating_a_Minimal_Bounding_Rectangle/links/5751a14108ae6807fafb2aa5.pdf
+ *
+ * @param points - Points from which to compute the MBR.
+ * @returns The array of corners.
+ */
+export function getMbrFromPoints(points: readonly Point[]): Mbr {
+  if (points.length === 0) {
+    return {
+      corners: [],
+      angle: 0,
+      width: 0,
+      height: 0,
+      surface: 0,
+      perimeter: 0,
+    };
+  }
+  if (points.length === 1) {
+    return {
+      corners: [points[0], points[0], points[0], points[0]],
+      perimeter: 0,
+      surface: 0,
+      angle: 0,
+      width: 0,
+      height: 0,
+    };
+  }
+
+  let rotatedVertices: Point[] = [];
+  let minSurface = Number.POSITIVE_INFINITY;
+  let minSurfaceAngle = 0;
+  let mbr: Point[] = [];
+
+  for (let i = 0; i < points.length; i++) {
+    let angle = getAngle(points[i], points[(i + 1) % points.length]);
+
+    rotatedVertices = rotate(-angle, points);
+    // console.log({ rotatedVertices });
+
+    // we rotate and translate so that this segment is at the bottom
```

Comment:
```suggestion
    // Rotate and translate so that this segment is at the bottom
```

Please avoid "we" in comments and documentation.